src/mame/drivers/8080bw.cpp

```diff
@@ -738,17 +745,18 @@ INPUT_PORTS_END
 static INPUT_PORTS_START( spaceat2 )
 	PORT_INCLUDE( sicv )
 
+	PORT_MODIFY("IN0")
+	PORT_DIPNAME( 0x04, 0x00, DEF_STR( Difficulty ) )   /* [code: 0x18ca-d1] */
+	PORT_DIPSETTING(    0x00, DEF_STR( Easy ) )
+	PORT_DIPSETTING(    0x04, DEF_STR( Hard ) )
+
 	PORT_MODIFY("IN2")
 	PORT_DIPUNKNOWN_DIPLOC( 0x10, 0x00, "SW1:5" )
 	PORT_DIPUNKNOWN_DIPLOC( 0x20, 0x00, "SW1:6" )
 	PORT_DIPUNKNOWN_DIPLOC( 0x40, 0x00, "SW1:7" )
 	PORT_DIPNAME( 0x80, 0x00, DEF_STR( Coinage ) )  PORT_DIPLOCATION("SW1:8")
 	PORT_DIPSETTING(    0x80, DEF_STR( 2C_1C ) )
 	PORT_DIPSETTING(    0x00, DEF_STR( 1C_1C ) )
-
-	/* Dummy port for cocktail mode (not used) */
-	PORT_MODIFY(INVADERS_CAB_TYPE_PORT_TAG)
-	PORT_BIT( 0xff, IP_ACTIVE_HIGH, IPT_UNKNOWN )
```

Comment:
This was intentionally removing the cabinet type machine configuration setting.  What’s the rationale for changing that?

Reply:
The game works fine in both cocktail and upright modes, I'm unaware if this is from the schematics of the actual board or what, as it's been in MAME since 0.121 (earliest on github)